### PR TITLE
feat: drpc attestation

### PR DESCRIPTION
**Pull request by wadeking98**
added support for drpc attestation

### CODE REVIEW

File: app/ios/AriesBifold.xcodeproj/project.pbxproj
Comment (jleach) [inline app/ios/AriesBifold.xcodeproj/project.pbxproj:1]: Did you mean to check this file in?

File: app/src/services/attestation.tsx [hunk 1]
```diff
@@ -413,6 +308,14 @@ export const AttestationProvider: React.FC<AttestationProviderParams> = ({ child
         .subscribe(handleOfferWithParam)
       setOfferSubscription(offerSub)
     }
+
+    while (true) {
```
Comment (jleach) [inline app/src/services/attestation.tsx]: Can we get stuck here forever?

Comment (wadeking98) [reply app/src/services/attestation.tsx]: it won't get stuck unless someone awaits the start method, but I see what you mean, I'll clean it up

**Review by jleach — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -57,9 +57,9 @@
     "@aries-framework/askar": "0.4.2",
     "@aries-framework/react-native": "0.4.2",
     "@aries-framework/indy-sdk-to-askar-migration": "0.4.2",
-    "@credo-ts/drpc": "patch:@credo-ts/drpc@npm%3A0.5.0-alpha.145#./.yarn/patches/@credo-ts-drpc-npm-0.5.0-alpha.145-4e1953647c.patch",
+    "@credo-ts/drpc": "patch:@credo-ts/drpc@npm%3A0.5.0-alpha.151#~/.yarn/patches/@credo-ts-drpc-npm-0.5.0-alpha.151-5bad8b38bd.patch",
```
Comment (cvarjao) [inline package.json]: seems odd that the patch path changed from `./.yarn/...` (relative to package.json directory) to `~/.yarn/...` (relative to user directory?)

Comment (wadeking98) [reply package.json]: hmm, ya good catch, I'll change it to the package.json directory

**Review by jleach — APPROVED** [review]

**Pull request merged**